acr-repositories.yaml

```diff
@@ -158,3 +158,7 @@ rules:
     ruleName: alpine
     repoName: library/alpine
     destinationRepo: imported/alpine
+  redis:
+    ruleName: redis
+    repoName: library/redis
+    destinationRepo: library/redis
```

Comment:
```suggestion
    destinationRepo: imported/redis
```

Reply:
Hi @davet1985 - could you move this so that destination is `imported/redis` please?